runtime/westend/src/constants.rs

```diff
@@ -33,7 +33,7 @@ pub mod time {
 	use primitives::v0::{Moment, BlockNumber};
 	pub const MILLISECS_PER_BLOCK: Moment = 6000;
 	pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
-	pub const EPOCH_DURATION_IN_SLOTS: BlockNumber = 1 * HOURS;
+	pub const EPOCH_DURATION_IN_SLOTS: BlockNumber = 1 * MINUTES;
```

Comment:
if this is intended, it should probably go into separate PR as it is orthogonal to staking miner